packages/expo-media-library/ios/EXMediaLibrary/EXMediaLibrary.m

```diff
@@ -401,25 +404,28 @@ - (NSDictionary *)constantsToExport
   }
   
   PHAsset *asset = [EXMediaLibrary _getAssetById:assetId];
+    
+  NSNumber* shouldDownloadFromNetwork = [options valueForKey:EXMediaLibraryShouldDownloadFromNetworkKey] ? : @1;
```

Comment:
```suggestion
BOOL shouldDownloadFromNetwork = [[options objectForKey:EXMediaLibraryShouldDownloadFromNetworkKey] boolValue] ?: @YES
```

Reply:
hmm if the value is `nil` then `[boolValue]` will crash, won't it? 🤔

Reply:
In `objective-c`, if you try to call a function on a `nil`, the program won't crash - it will return `nil` instead.

Reply:
See https://stackoverflow.com/a/2696909/13361748 ;)

Reply:
Oh nice 😄 , TIL